morello/morello.md

```diff
@@ -159,6 +159,10 @@ All content in this document is at the **Alpha** quality level.
 
 * Update the copyright statement in section [Copyright](#copyright).
 * Add a reference to the [CHERI-HYBRID](https://github.com/CTSRD-CHERI/cheri-hybrid-c-guide) *CHERI Hybrid C/C++ Programming Guide* in section [Scope](#scope).
+* Converted document sources from reStructuredText (`.rst`) to
+  Markdown (`.md`). The [`pandoc`](https://pandoc.org/) is now used to
```

Comment:
```suggestion
  Markdown (`.md`). [`Pandoc`](https://pandoc.org/) is now used to
```
This suggestion applies for every instance this change is described. Alternatively, "The pandoc command" should also work.

Reply:
I opted for `The tool pandoc ...`